src/modules/graphics/graphics.c

```diff
@@ -4027,14 +4050,15 @@ void lovrPassTock(Pass* pass, Tally* tally, uint32_t index) {
 // Helpers
 
 static void* tempAlloc(size_t size) {
+  lovrCheck(size < SIZE_MAX-MAX_FRAME_MEMORY, "Allocation over " MAX_FRAME_MEMORY_STR);
```

Comment:
Although our error messages aren't great, I think they should be slightly nicer than "Shader too big" and "Allocation over 1GiB".  Also, binary operators should have spaces around them.  If possible this check should be combined into the one below, or replace it.